CI failing on macOS

```
Run coverallsapp/github-action@648a8eb78e6d50909eff900e4ec85cab4524a45b
Run echo "Warning: The coverage-reporter-platform parameter is not available on macOS. The default version for macOS will be used." >&2
Warning: The coverage-reporter-platform parameter is not available on macOS. The default version for macOS will be used.
Run # Enable debugging if 'debug' is true
Error: Refusing to load formula coverallsapp/coveralls/coveralls from untrusted tap coverallsapp/coveralls.
Run `brew trust --formula coverallsapp/coveralls/coveralls` or `brew trust coverallsapp/coveralls` to trust it.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -37,6 +37,10 @@ jobs:
         if: runner.os == 'macOS'
         run: npm test
 
+      - name: Trust coveralls (macOS)
+        if: runner.os == 'macOS'
+        run: brew trust coverallsapp/coveralls
+
       - name: Upload coverage
         uses: coverallsapp/github-action@648a8eb78e6d50909eff900e4ec85cab4524a45b # v2.3.6
         with:

